testing/btest/coverage/bare-load-baseline.test

```diff
@@ -9,6 +9,7 @@
 # below does. Don't ask. :-)
 
 # @TEST-REQUIRES: $SCRIPTS/have-spicy  # This test logs loaded scripts, so disable it if Spicy and it associated plugin is unavailable.
+# @TEST-REQUIRES: ! grep -q "#define ENABLE_SPICY_SSL" $BUILD/zeek-config.h
```

Comment:
Can you add a comment why we don't run this with the Spicy SSL analyzer?

Reply:
Will do. I just was too lazy to add comments for the more-obvious-testcases, that just change on every major change in Zeek.